firestore-bigquery-export/extension.yaml

```diff
@@ -228,13 +251,14 @@ params:
     default: posts
     required: true
 
-  - param: TABLE_PARTITIONING
-    label: BigQuery SQL table partitioning option
+
+  - param: TIME_PARTITIONING
+    label: BigQuery SQL table Time Partitioning option type
     description: >-
-      This parameter will allow you to partition the BigQuery table and BigQuery view 
+      This parameter will allow you to partition type of the BigQuery table and BigQuery view 
       created by the extension based on data ingestion time. You may select the granularity of
       partitioning based upon one of: HOUR, DAY, MONTH, YEAR. This will
-      generate one partition per day, hour, month or year, respectively. 
+      generate one partition per day, hour, month or year, respectively. The table is partitioned by pseudo column '_PARTITIONTIME' or you can be assign it to BigQuery Table schema field if other parameters passed.
```

Comment:
Update probably isn't needed here, the original description just covers this field only?